<ui version="4.0" >
 <class>dlgErrorForm</class>
 <widget class="QDialog" name="dlgErrorForm" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize" >
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string> </string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>Images/error.png</normaloff>Images/error.png</iconset>
  </property>
  <property name="windowFilePath" >
   <string> </string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="topMargin" >
    <number>6</number>
   </property>
   <property name="bottomMargin" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="layoutErrorMessage" >
     <item>
      <widget class="QFrame" name="frame" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth" >
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblImage" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="pixmap" >
           <pixmap>Images/big_error.png</pixmap>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblText" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize" >
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" name="layoutShowDetails" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbnShowDetails" >
       <property name="text" >
        <string>Show Details...</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk" >
       <property name="text" >
        <string>Ok</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QScrollArea" name="saDetails" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable" >
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>457</width>
        <height>104</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QTextEdit" name="lblDetailText" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
